11-node-frameworks/KononovichGalina/index.js

```diff
@@ -0,0 +1,55 @@
+const koa = require('koa');
+const router = require('koa-router');
+const serve = require('koa-static');
+const app = new koa();
+const session = require('koa-generic-session');
+const init = router();
+// const KoaBody = require('koa-body');
+
+// koaBody = convert(KoaBody());
+
+// init.get('/get', getData);
+
+// function getData(ctx, next) {
+//   console.log('nfj');
+//   ctx.body = 'get data';
+//   next();
+// }
+
+// app.use(init.routes());
+
+
+app.use(serve(__dirname + '/public'));
+
+app.use(async(ctx, next) => {
```

Comment:
What purpose for this middleware and the next one in terms of working application?